src/push-notifications/use-register.tsx

```diff
@@ -0,0 +1,9 @@
+import {registerForPushNotifications} from '@atb/push-notifications/api';
+import {useMutation} from '@tanstack/react-query';
+
+export const useRegister = () => {
```

Comment:
Could consider renaming to `useRegisterForPushNotifications` or `useNotificationRegister`